scripts/full-tutorial-run.sh

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env bash
+
+set -ex
+set -o pipefail
+
+# AMI SETUP START #
+
+export MAKEFLAGS=-j16
+CY_COMMIT=9b3fef
+
+################################### CHIPYARD AM
+cd ~/
+
+(
+# get a subshell so can source sourceme-f1-manager.sh later
+
+# get chipyard
+git clone https://github.com/ucb-bar/chipyard chipyard-morning
+cd chipyard-morning
+git checkout $CY_COMMIT
```

Comment:
Shouldn't this checkout the latest chipyard? If the idea is that tutorials can just use the latest master.

Reply:
I think that is should check out the version of Chipyard that is committed in FireSim. If it just tracks Chipyard main then it could randomly fail on a PR that has nothing to do with Chipyard.

Reply:
For now, this is hardcoded so I can test things. I'll remove this before the final merge.